test_rendering/spec/ol/map.test.js

```diff
@@ -16,6 +16,7 @@ describe('ol.rendering.Map', function() {
     target = createMapDiv(50, 50);
 
     var vectorLayer = new ol.layer.Vector({
+      zIndex: 0,
```

Comment:
I addressed this in [Commit-b843d7d](https://github.com/openlayers/openlayers/pull/6661/commits/b843d7de52e4ea89746c05dbdef2b8f31ea57293)